apps/settings/js/modules/addon_manager.js

```diff
@@ -51,7 +51,16 @@ define(function(require) {
       // manifest, we need to listen to 'downloadsuccess' on the app
       // (not mozApps.mgmt), and that's when manifest will be ready.
       AppsCache.addEventListener('oninstall', (evt) => {
-        evt.application.addEventListener('downloadsuccess', _boundUpdateAddons);
+        var app = evt && evt.application;
+        // Bug 1220195
+        // If installed via WebIDE, _isAddon will return true, and we
```

Comment:
nit: can you expand you comment to mention that downloadsuccess never fires for the webide installed addons.